hieradata/staging.yaml

```diff
@@ -4,7 +4,7 @@ app_domain: 'staging.publishing.service.gov.uk'
 backup::server::backup_hour: 9
 
 base::shell::shell_prompt_string: 'staging'
-base::supported_kernel::enabled: true
+base::supported_kernel::enabled: false
```

Comment:
In the class this is set to `false` by default, so I think these can just be removed unless we want to keep them for clarify.

Reply:
I was keeping them for clarity. We'll need to do a few PRs in succession to change these, so I figured it'd make it easier to see what needs changing